src/Utility/AxonUtils.js

```diff
@@ -248,6 +248,7 @@ class AxonUtils {
      * @param {String|MessageObject} content - String or object (embed)
      * @param {Object} [options={}] - Options { disableEveryone: Boolean, delete: Boolean, delay: Number }
      * @param {Boolean} [options.disableEveryone=true] - Whether to allow mentioning everyone or not
+     * @param {Object} [options.allowedMentions] - Cutom allowed mentions object
```

Comment:
Properties plus typo (same as before)
```suggestion
     * @param {Object} [options.allowedMentions] - Custom allowed mentions object
```